docs/manuals/source/Appendix/Checkpoints.rst

```diff
@@ -0,0 +1,18 @@
+.. _checkpoints-chapter:
+
+Checkpoints
+===========
+
+Checkpoints are a way to save the progress of a backup while it is running.
+
+When a checkpoint is executed, the files that were successfully stored in the media at that point in time will be updated in the catalog database.
+
+Checkpoints happen after a certain amount of time set by the user using the :config:option:`storage/storage/CheckpointInterval` directive.
+
+Checkpoints also happen on volume changes. This means that when a volume is full, or the backup has to switch to writing to another volume for some other reason, a checkpoint is triggered saving what has been written to that volume.
+
```

Comment:
Based on the Checkpoints Feature, a resume of cancelled or broken Backupjobs is conceivable in the future.